### PR TITLE
Introduce Boston recommendations

### LINKED ISSUE #11: Add recommendation file for Boston
- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a Pull Request
- [ ] Review
- [ ] Merge

### PR DESCRIPTION
**Pull request by JudTown17**
Closes #11.

### CODE REVIEW

**Comment by eironside** [comment]
Love Beantown

**Comment by trevorjiang** [comment]
Nice code 👍

File: JudTown17-Boston.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Boston
+
+### Places to Eat
+
+- Dunkin Donuts
```
Comment (eironside) [inline JudTown17-Boston.md:5]: Chocolate cake doughnuts are the best.  Does America really run on Dunkin?

Comment (trevorjiang) [inline JudTown17-Boston.md:5]: I like this too.  Do u have any other dessert ?

Comment (ENOONE) [reply JudTown17-Boston.md:5]: @eironside No idea.

**Review by eironside — DISMISSED** [review]
How about sushi?

**Comment by eironside** [comment]
good

**Review by eironside — COMMENTED** [review]
good

**Pull request merged**